.github/workflows/manual-add-label.yml

```diff
@@ -0,0 +1,45 @@
+name: "Manual: Apply Labels in Bulk"
+
+on: 
+  workflow_dispatch:
+    inputs:
+      issues:
+        description: a single or comma separated list of issue numbers
+        required: true
+        type: string
+      labels:
+        description: a single or comma separated list of labels to apply to all listed issues
+        required: true
+        type: string
+      colors:
+        description: |
+          A single or comma separated list of colors to create the labels with if needed.
+          the list order is the same as `labels`. Missing or blank values (e.g. `FFFFFF,,FFFFFF`) use the `default_color`
+        type: string

```

Comment:
Especially creating them on demand and with specific colors. I *really* doubt we will ever be both applying *and* creating labels en-masse like this